docs/install/common-problems.md

```diff
@@ -98,3 +98,29 @@ To correct this issue:
   - Redo all necessary [installation instructions in **steps 1 through 6**](install-cave.md#download-and-installation-instructions_2)
 
 ---
+
+## Troubleshooting Uninstalling EDEX
+
+Sometimes yum can get in a weird state and not know what AWIPS groups have been installed. For example if you are trying to remove AWIPS you may see an error:
```

Comment:
I might just add that you're specifically talking about Linux.  (I know yum is only a linux thing, but it still might be more clear to some of our other users?)